apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -73,11 +123,15 @@ function WpcomBlockEditorNavSidebar() {
 		return null;
 	}
 
-	let closeUrl = addQueryArgs( 'edit.php', { post_type: postType.slug } );
-	closeUrl = applyFilters( 'a8c.WpcomBlockEditorNavSidebar.closeUrl', closeUrl );
+	const defaultCloseUrl = addQueryArgs( 'edit.php', { post_type: postType.slug } );
+	const closeUrl = applyFilters( 'a8c.WpcomBlockEditorNavSidebar.closeUrl', defaultCloseUrl );
 
-	let closeLabel = get( postType, [ 'labels', 'all_items' ], __( 'Back', 'full-site-editing' ) );
-	closeLabel = applyFilters( 'a8c.WpcomBlockEditorNavSidebar.closeLabel', closeLabel );
+	const defaultCloseLabel = get(
+		postType,
+		[ 'labels', 'all_items' ],
+		__( 'Back', 'full-site-editing' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Back', 'NULL', { context: 'NULL'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/nav-sidebar-style)